docs/documentation/introduction.rst

```diff
@@ -0,0 +1,387 @@
+.. _introduction:
+
+Introduction
+============
+
+.. contents::
+    :local:
```

Comment:
I think we should consider not using local TOC on pages like this. The only real point to this page is to be read linearly anyway and the TOC just gets in the way.